src/services/versionService/openapi.yaml

```
@@ -0,0 +1,80 @@
+security:
+  - {}
+info:
+  title: HPI-Schul-Cloud version Service API
+  description:
+    This is the API specification for the HPI-Schul-Cloud version service.
+    
+  contact:
+    name: support
+    email: info@hpi-schul-cloud.de
+  license:
+    name: GPL-3.0
+    url: 'https://github.com/hpi-schul-cloud/schulcloud-server/blob/master/LICENSE'
+  version: 1.0.0
+
+components:
+  schemas:
+    migration:
+      type: object
+      properties:
+        _id:
+          type: string
+          pattern: '^[0-9a-f]{24}$'
+        state:
+          type: string
+          enum: [up, down]
+        name:
+          type: string
+        createdAt:
+          type: string
+          format: date-time
+        __v:
+          type: integer
+    version:
+      type: object
+      properties:
+        sha:
+          type: string
```

The git hash should be a Sha1 and 40 digits long. Maybe we should encode this as pattern?

```suggestion
          type: string
          pattern: '^[0-9a-f]{40}$'
```